manifests/o/OBSProject/OBSStudio/27.2.1/OBSProject.OBSStudio.locale.en-US.yaml

```diff
@@ -0,0 +1,166 @@
+
+  • Fixed a bug where certain settings would show as available in the video capture device properties when they weren’t actually available [lexi-lambda]
+
+  • Fixed a crash where modifying the video capture device properties on Linux could cause a crash on some systems [lexi-lambda]
+
+  • Fixed Linux PulseAudio monitoring volume with signed 32bit and unsigned 8bit formats [norihiro]
+
+  • Fixed Media Source not working with audio-only files [norihiro]
+
+  • Fixed a couple memory leaks with QSV [shleeable]
+
+  • Fixed an issue where audio monitoring could have significant latency on Linux [kkartaltepe]
+
+  • Fixed undo/redo not working for text changes on the context bar [gxalpha]
+
+  • Fixed an issue where the transform window would not consistently updated for selected sources/scenes [offthegrid-mike]
+
+  • Fixed missing undo/redo action when resetting filter properties [gxalpha]
+
+  • Fixed missing undo/redo action when using “Add existing source” [gxalpha]
+
+  • Fixed an issue where stinger transitions could sometimes loop unintentionally [exeldro]
+
+  • Fixed an issue where transforms/crops would not be included when copying/pasting sources [torresam]
+
+  • Fixed a couple potential freezes and crashes, especially with browser sources [Jim]
+
+  • Fixed an issue where devices would not always reinitialize properly when switching scene collections [Jim]
+
+  • Fixed various issues that would affect the websockets plugin [tt2468]
+
+  • Fixed a bug where certain Twitch panel settings wouldn’t save properly [derrod]
+
+  • Fixed a bug on Linux where Window capture wouldn’t properly capture on software rasterization devices [ishitatsuyuki]
+
+  • Fixed various potential bugs and race conditions across the program and multiple plugins [wangshaohui]
+
+  • Fixed network throughput to distant servers potentially being limited in low memory environments [R1CH]
+
+  • Fixed a crash on Linux X11 that could happen when closing certain dialogs containing video previews [tt2468]
+
+  • Fixed VST plugin windows disappearing when resized [Blue Cat Audio]
+
+  • Fixed a VST plugin crash that could occur under certain circumstances [walker-WSH]
+
+  • Fixed an incorrect offset of scene labels on certain multiview configurations [norihiro]
+
+  • Fixed a bug that would allow resizing sources that are locked [ntoff]
+
+  • Fixed a performance issue with X11 window capture on Linux [ishitatsuyuki]
+
+  • Fixed a bug where projectors could not be resized with certain window managers on Linux [univrsal]
+
+  • Fixed window flashing on macOS when starting the program minimized to tray [CodyJung]
+
+  • Fixed a bug where push-to-talk or mute delay would not save under certain circumstances [CodyJung]
+
+  • Fixed a bug where mouse hotkeys would not always function correctly on Linux [kkartaltepe]
+ReleaseNotesUrl: https://github.com/obsproject/obs-studio/releases/tag/27.2.0

```

Comment:
Hey, just for later, please check if these fields are defined

Reply:
Done, manually updated those, release notes.